src/commons/sideContent/SideContentTypes.ts

```diff
@@ -24,6 +24,7 @@ export enum SideContentType {
   grading = 'grading',
   introduction = 'introduction',
   inspector = 'inspector',
+  githubAssessments = 'githubAssessments',
```

Comment:
this is still unused in this pr

Reply:
type removed:
https://github.com/chekjun/cadet-frontend/blob/master/src/commons/sideContent/SideContentTypes.ts